utxo/utxo.go

```diff
@@ -1913,6 +1914,50 @@ func (uv *UtxoVM) queryContractMethodACLWithConfirmed(contractName string, metho
 	return uv.aclMgr.GetContractMethodACLWithConfirmed(contractName, methodName)
 }
 
+func (uv *UtxoVM) queryTxFromForbiddenWithConfirmed(txid []byte) (bool, bool, error) {
+	request := &pb.InvokeRequest{
+		ModuleName:   "wasm",
+		ContractName: "forbidden",
+		MethodName:   "get",
+		Args: map[string][]byte{
+			"txid": []byte(fmt.Sprintf("%x", txid)),
+		},
+	}
+	modelCache, err := xmodel.NewXModelCache(uv.GetXModel(), true)
+	if err != nil {
+		return false, false, err
+	}
+	contextConfig := &contract.ContextConfig{
+		XMCache:        modelCache,
+		ResourceLimits: contract.MaxLimits,
+	}
+	moduleName := request.GetModuleName()
+	vm, err := uv.vmMgr3.GetVM(moduleName)
+	if err != nil {
+		return false, false, err
+	}
+	contextConfig.ContractName = request.GetContractName()
+	ctx, err := vm.NewContext(contextConfig)
+	if err != nil {
+		return false, false, err
+	}
+	_, err = ctx.Invoke(request.GetMethodName(), request.GetArgs())
+	if err != nil {
+		return false, false, err
+	}
+	ctx.Release()
+	// inputs as []*xmodel_pb.VersionedData
+	inputs, _, _ := modelCache.GetRWSets()
+	versionData := &xmodel_pb.VersionedData{}
+	if len(inputs) == 2 {
+		versionData = inputs[1]
+	} else {
```

Comment:
to avoid unnecessary "else" which may cause code nesting, it's better to check "!=" and return.

Reply:
I will fix it.